lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -5,6 +5,22 @@
     </a>
 <% }else{ %>
     <div class="enrollment-info"><%- gettext('not enrolled') %></div>
+    <% if (run_modes.length > 1){ %>
+        <div class="run-select-container">
+            <label class="sr-only" for="select-<%- course_key %>-run">Select Course Run</label>
+            <select id="select-<%- course_key %>-run" class="run-select">
+                <% _.each (run_modes, function(runMode){ %>
+                    <option value="<%- runMode.run_key %>">
+                        <%= interpolate(
+                            gettext('Starts %(start)s'),
+                            { start: runMode.start_date },
+                            true)
+                        %>
+                    </option>
+                <% }); %>
+            </select>
+        </div>
+    <% } %>
```

Comment:
My understanding of the requirement is that if there is only 1 course run that you still display the start data, but it's just not a form field. Was there a wireframe confirming the desired UI for single run enrollment?